sympy/integrals/integrals.py

```diff
@@ -532,6 +533,38 @@ def try_meijerg(function, xab):
                             function = ret
                             continue
 
+            if not isinstance(antideriv, Integral) and antideriv is not None:
+                sym = xab[0]
+                if antideriv.is_Add:
+                    terms = antideriv.args
+                else:
+                    terms = tuple((antideriv,))
+                for term in terms:
+                    atan_part = term.atoms(atan)
```

Comment:
Would it be possible to consider `atan_terms in antideriv.atoms(atan)` instead of considering each `term` of an `Add` separately?

Reply:
yes in the end it is taking every `atan` term. So it's better to use `atan_terms in antideriv.atoms(atan)`